Sources/MapboxDirections/Intersection.swift

```diff
@@ -319,8 +323,9 @@ extension Intersection: Codable {
             approachLanes = lanes.map { $0.indications }
             usableApproachLanes = lanes.indices { $0.isValid }
             preferredApproachLanes = lanes.indices { ($0.isActive ?? false) }
-            let usableIndications = lanes.compactMap { $0.validIndication }
-            usableLaneIndication = usableIndications.reduce(LaneIndication(rawValue: 0)) { $0.union($1) }
+            usableLaneIndication = lanes.compactMap { $0.validIndication }[0]
```

Comment:
Some tests are [crashing](https://app.circleci.com/pipelines/github/mapbox/mapbox-directions-swift/997/workflows/ed18f821-8a90-402f-85cc-c1a89f5f70b6/jobs/3045) because of indexing into an empty array:

```
Fatal error: Index out of range: file Swift/ContiguousArrayBuffer.swift, line 444
2021-04-12 14:40:33.761822+0000 xctest[3358:53526] Fatal error: Index out of range: file Swift/ContiguousArrayBuffer.swift, line 444
```

Use `Array.first` to avoid the crash.